doc/users/prev_whats_new/whats_new_3.5.0.rst

```diff
@@ -179,6 +253,14 @@ argument, for example:
     ax.legend(title='Points',
               title_fontproperties={'family': 'serif', 'size': 20})
 
+``Text`` and ``TextBox`` added *parse_math* option
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
```suggestion
--------------------------------------------------
```
Change to dashes here so the indentation is at the same level as all of the other items. Also above in the "A callback registry..." item.

Reply:
Done.

Reply:
👍  Feel free to self-merge after CI if you've got all the what's new entries you want in this one.